go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/cortexproject/cortex
 
 go 1.21
```

Comment:
```suggestion
go 1.21.0
```

Reply:
https://github.com/cortexproject/cortex/pull/6072 this is definitely a better option